File: strings/strings.ts
function main() {
  // String concatenation
  const str1: string = "Hello";
  const str2: string = "World";
  const result: string = str1 + " " + str2;
  console.log(result);

  // String length
  const str: string = "Hello, World!";
  const length: number = str.length;
  console.log("Length:", length);

  // String splitting
  const split: string[] = str.split(" ");
  console.log("Split result:", split);

  // String substring
  const substr: string = str.substring(0, 5);
  console.log("Substring:", substr);

  // String contains
  const contains: boolean = str.includes("World");
  console.log("Contains:", contains);
}

main();

File: strings/strings.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	// String concatenation
	str1 := "Hello"
	str2 := "World"
	result := str1 + " " + str2
	fmt.Println(result)

	// String length
	str := "Hello, World!"
	length := len(str)
	fmt.Println("Length:", length)

	// String splitting
	str = "Welcome to GoLang"
	split := strings.Split(str, " ")
	fmt.Println("Split result:", split)

	// String substring
	str = "Hello, World!"
	substr := str[0:5]
	fmt.Println("Substring:", substr)

	// String contains
	str = "Hello, World!"
	contains := strings.Contains(str, "World")
	fmt.Println("Contains:", contains)
}
